python/src/multi_agent_orchestrator/agents/anthropic_agent.py

```diff
@@ -263,23 +259,19 @@ async def process_request(
 
         messages = self._prepare_conversation(input_text, chat_history)
         system_prompt = await self._prepare_system_prompt(input_text)
-        input = self._build_input(messages, system_prompt)
+        json_input = self._build_input(messages, system_prompt)
 
-        return await self._process_with_strategy(self.streaming, input, messages)
+        return await self._process_with_strategy(self.streaming, json_input, messages)
 
     async def handle_single_response(self, input_data: dict) -> Any:
         try:
-            response = self.client.messages.create(**input_data)
-            return response
+            return self.client.messages.create(**input_data)
         except Exception as error:
             Logger.error(f"Error invoking Anthropic: {error}")
             raise error
 
-    async def handle_streaming_response(self, input) -> AsyncGenerator[AgentStreamResponse, None]:
-        message = {}
-        content = []
-        accumulated = {}
-        message['content'] = content
+    async def handle_streaming_response(self, input) -> AsyncGenerator[AgentStreamResponse, None]: # noqa: A002
```

Comment:
Adding `# noqa: A002` skip the Ruff validation for this rule. Since this is a "public" function, idk if I can change the signature? WDYT?